portal-api/src/v1/controllers/utilisation-report-service/utilisation-report-upload.controller.js

```diff
@@ -49,16 +49,20 @@ const sendEmailToPdcInputtersEmail = async (bankName, reportPeriod) => {
 const sendEmailToBankPaymentOfficerTeam = async (reportPeriod, bankId, submittedDate, user) => {
   try {
     const reportSubmittedBy = `${user.firstname} ${user.surname}`;
-    const { teamName, email } = await getPaymentOfficerTeamDetailsFromBank(bankId);
+    const { teamName, emails } = await getPaymentOfficerTeamDetailsFromBank(bankId);
     const formattedSubmittedDate = formatDateForEmail(submittedDate);
 
-    await sendEmail(EMAIL_TEMPLATE_IDS.UTILISATION_REPORT_CONFIRMATION, email, {
-      recipient: teamName,
-      reportPeriod,
-      reportSubmittedBy,
-      reportSubmittedDate: formattedSubmittedDate,
-    });
-    return { paymentOfficerEmail: email };
+    await Promise.all(
+      emails.map(async (email) => {
+        await sendEmail(EMAIL_TEMPLATE_IDS.UTILISATION_REPORT_CONFIRMATION, email, {
```

Comment:
For this one, we can just directly return the `Promise` returned by `sendEmail` rather than explicitly `await`ing it:
```suggestion
      emails.map((email) => {
        sendEmail(EMAIL_TEMPLATE_IDS.UTILISATION_REPORT_CONFIRMATION, email, {
```

Reply:
Ahh, I don't explicitly return anything so telling me to use `forEach`

Reply:
If you remove the opening curly bracket that lint rule should go away I think :)